### ISSUE TITLE
Random/intermittent "Unknown interaction" `10062`

### ISSUE THREAD

**Issue by hippietrail** [hippietrail/hippiebot.js#18]
This has been happening since I got back to Laos from Thailand.  
Wi-Fi is typically slower here as I've seen many timeouts in the `/yt` YouTube command. So this may be related to that.  
It seems to often but not always happen when doing autocomplete, which is only used in the `/yt` command.  
I believe it's due to something getting out of sync in interactions, like failing to respond to one, or responding twice, but I can't see anywhere that would happen. Perhaps a deep/hidden exception that evades my `catch`es.
```json
[ERROR] {
  "requestBody": {
    "json": {
      "type": 5,
      "data": {}
    }
  },
  "rawError": {
    "message": "Unknown interaction",
    "code": 10062
  },
  "code": 10062,
  "status": 404,
  "method": "POST",
  "url": "https://discord.com/api/v10/interactions/1208648636951695410/aW50ZXJhY3Rpb246MTIwODY0ODYzNjk1MTY5NTQxMDpIeG9KWlpHSnlqV1JHNVA5ZWprUWM1T0dBNXFBRko3MmNyUWpTQVNoUzMyNE4xR3dnb0hvUWtRUklmNTdqMzlkb0kxSkNoTWNuZThsVVh5VmF4bGl4RGdqdjdKc3dhNU1pc29rT1JRMUdEZThhS3ZOWVdDZGduTElPY2VuckZKQw/callback"
}
```

**Comment by hippietrail** [comment]
It's not directly related to autocomplete. Even if I remove autocomplete support it can still happen. I think it's related to `/yt` somehow though.

**Comment by hippietrail** [comment]
OK it's been a while and this was definitely fixed by [7261370](https://github.com/hippietrail/hippiebot.js/commit/7261370fd34357fd2a2393e53cdfff785edf6871)
